### ISSUE TITLE
[Accessibility][ChoiceGroup/ChoiceField]: Component should have aria-disabled attribute set when it is disabled

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1084]
For disabled choice group and field, aria-disabled attribute should be specified, else screen readers will have trouble reading it clearly. See this: http://www.w3.org/TR/wai-aria/states_and_properties#aria-disabled

Repro:
https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/choicegroup

The html for disabled choicefield is as below:
`<div class="ms-ChoiceField"><input type="radio" id="ChoiceGroup1846-C" class="ms-ChoiceField-input" name="ChoiceGroup1846" disabled="" aria-describedby="ChoiceGroupDescription1847-C" value="on"><label for="ChoiceGroup1846-C" class="ms-ChoiceField-field is-disabled"><span id="ChoiceGroupDescription1847-C" class="ms-Label">Option C</span></label></div>`

Group level disabled should also be handled.

**Comment by mdahamiwal** [comment]
disabled elements shouldn't be focussable unless part of a menu or a design pattern clearly specifies it. In case of checkboxes we are using disabled attribute that should prevent focus on these elements, thus we don't need an explicit aria-disabled on these. Does that answer the concern here?

**Comment by satishc2437** [comment]
Hi,
As per the ARIA-Practices, the property needs to be set irrespective of the element getting focus or not. Below is a snap from the ARIA page (http://www.w3.org/TR/wai-aria/states_and_properties#aria-disabled)

![image](https://cloud.githubusercontent.com/assets/13657916/23251149/32754b30-f9d2-11e6-9b55-6462fe30888f.png)

Based on this, I feel we should set the aria-disabled attribute along with disabled tag.

Thanks,
Satish

**Comment by aleksandrjPersonal** [comment]
Merged March 9th
